frontend/src/components/__tests__/Button.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import 'jest';
+
+import Button from '../shared_components/Button';
+
+describe('<Button />', () => {
+    it('renders without crashing', () => {
+      const div = document.createElement('div');

```

Comment:
https://github.com/Lambda-School-Labs/labs9-cleaner-pos/blob/b412a9d026a11570e2dd976d60968a0b49c13442/frontend/src/components/__tests__/Button.spec.tsx

I've created a Button test that should give you a rough idea of how the tests should be structured.
It's not complete yet, we'd still need to confirm that the onclick works and so on (also the component needs to support the desired spec) 
Future tests should mirror this roughly 👍